multimap/src/main/java/org/infinispan/multimap/impl/EmbeddedMultimapPairCache.java

```diff
@@ -83,7 +83,7 @@ public CompletionStage<Map<HK, HV>> get(K key) {
       requireNonNull(key, ERR_KEY_CAN_T_BE_NULL);
       return cache.getCacheEntryAsync(key)
             .thenApply(entry -> {
-               if (entry == null) {
+               if (entry == null || entry.getValue() == null) {
```

Comment:
How can this be null? The only cache that allowed null values afaik is the scattered cache but that has been removed.

Reply:
You're right. It was a mistake on the new ReadWrite impl. I was accepting entries that should not be added to the data container.